src/footer/footer.html

```diff
@@ -25,39 +29,69 @@
       <div class="section launch">
         <div class="title">Prime Launch</div>
         <div class="body">
-          <a href="#" target="_blank" rel="noopener noreferrer">Why Prime Launch?</a>
-          <a href="#" target="_blank" rel="noopener noreferrer">Upcoming Launches</a>
-          <a href="#" target="_blank" rel="noopener noreferrer">Host a Launch</a>
-          <a href="#" target="_blank" rel="noopener noreferrer">Prime Launch Docs</a>
+          <a href="#" target="_blank" rel="noopener noreferrer"
+            >Why Prime Launch?</a
```

Comment:
All of these kinds of changes are really unnecessary and in this case, are cluttering up the code.  They make it unnecessarily harder for reviewers to do their job.

Best practice is that we review our own PRs and remove unnecessary stuff like this, before inviting in reviewers...